pkg/scheduler/api/types.go

```diff
@@ -200,6 +200,9 @@ type ExtenderConfig struct {
 	//   will skip checking the resource in predicates.
 	// +optional
 	ManagedResources []ExtenderManagedResource
+	// Ignorable specifies if the extender is ignorable, i.e. scheduling should not
+	// fail when this extender is unavailable.
```

Comment:
Let us the change the comment as well to something like "scheduling should not fail when extender returns an error"